code/game/objects/items/weapons/cards_ids.dm

```diff
@@ -222,7 +222,6 @@ var/global/const/NO_EMAG_ACT = -50
 		send_rsc(user, side, "side.png")
 	var/datum/browser/written/popup = new(user, "idcard", name, 600, 250)
 	popup.set_content(dat())
-	popup.set_title_image(usr.browse_rsc_icon(src.icon, src.icon_state))
```

Comment:
Why are these being removed?

Reply:
Legacy code which doesn't even work.

Reply:
Fair enough, I guess. You should probably remove `proc/set_title_image` in that case.